includes/reports/overview/list-table/class-sensei-reports-overview-list-table-students.php

```diff
@@ -222,4 +212,25 @@ private function format_date_registered( string $date ) {
 
 		return wp_date( get_option( 'date_format' ), $date->getTimestamp(), $timezone );
 	}
+
+	/**
+	 * Format user name.
+	 *
+	 * @param int $user_id user's id.
+	 * @param int $user_name user's name.
+	 *
+	 * @return string Formatted user name to the HTML table.
+	 */
+	private function format_user_name( $user_id, $user_name ) {
```

Comment:
As an option, we could check `$csv_output` here and return `$user_name` if it is true.
It will allow us to remove [the ternary from column data](https://github.com/Automattic/sensei/pull/5096/files#diff-2e9e12b93ce5b958ab9f40e4b4f87efb512ccd8bd60184c7228141a6476b6638R160).

Reply:
@merkushin Done here 87cc4c43abeb1dd72cc63f1a65af2f589a3d0340